src/query/ast/src/parser/statement.rs

```diff
@@ -151,6 +151,37 @@ pub fn statement(i: Input) -> IResult<StatementMsg> {
         },
     );
 
+    let merge = map(
+        rule! {
+            MERGE ~ #hint? ~ INTO ~ #period_separated_idents_1_to_3  ~ #table_alias? ~ USING
+            ~ #merge_source  ~ ON ~ #expr ~ (#match_clause | #unmatch_clause)*
+        },
+        |(
+            _,
+            opt_hints,
+            _,
+            (catalog, database, table),
+            alias_target,
+            _,
+            source,
+            _,
+            join_expr,
+            merge_options,
+        )| {
+            // we can get the target_table_reference in later, it doesn't matter.
```

Comment:
what does this comment mean?

Reply:
The TableReference Will generate in bind_merge_into. Let me remove this.